.gitignore

```diff
@@ -35,3 +35,5 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+.idea
```

Comment:
You must this into your global gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore